workflow/metrics/metrics_test.go

```diff
@@ -66,6 +66,11 @@ func TestMetrics(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, float64(0), *metric.Gauge.Value)
 
+	m.OperationCompleted(0.05)
+	err = m.operationDurations.Write(&metric)
+	assert.Nil(t, err)
```

Comment:
I think we should be making a habit of 

1. always using NoError
2. typically using 


if assert.NoError(..) {
// only then check on nillable variables, to prevent panic
}

Reply:
Done!